src/main/scala/io/viash/functionality/dependencies/Dependency.scala

```diff
@@ -32,15 +32,15 @@ import io.viash.exceptions.MissingBuildYamlException
   """name: qc/multiqc
     |  repository: 
     |    type: github
-    |    uri: openpipelines-bio/modules
+    |    repo: openpipelines-bio/modules
     |    tag: 0.3.0
     |""".stripMargin,
   "yaml",
   "Definition of dependency with a fully defined repository"
 )
 @exampleWithDescription(
   """name: qc/multiqc
-    |repository: "github://openpipelines-bio/modules:0.3.0"
+    |  repository: "github://openpipelines-bio/modules:0.3.0"
```

Comment:
This indenting seems strange. I'd expect

```
  """name: qc/multiqc
    |repository: "github://openpipelines-bio/modules:0.3.0"
```

Or 
```
  """  name: qc/multiqc
    |  repository: "github://openpipelines-bio/modules:0.3.0"
```

Reply:
I'm not sure what went wrong but the first would be the one I expect too, aka how it was before.
There are multiple mistakes in the other examples though.